src/main/java/com/splunk/logging/AckManager.java

```diff
@@ -39,26 +39,52 @@
 
   private static final ObjectMapper mapper = new ObjectMapper();
   private final HttpEventCollectorSender sender;
-  private final AckPollScheduler ackPollController = new AckPollScheduler();
+  private final PollScheduler ackPollController = new PollScheduler();
```

Comment:
Can we name the variables to be consistent with the class names? aka PollScheduler ackPollScheduler.